pkg/tsdb/legacydata/conversions.go

```diff
@@ -0,0 +1,85 @@
+package legacydata
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+
+	"github.com/grafana/grafana/pkg/apis/query/v0alpha1"
+)
+
+// ToDataSourceQueries returns queries that should be sent to a single datasource
```

Comment:
Nit. I would put these helpers under apis somewhere. legacydata package here refers to legacy query contracts used before the SDK contracts was introduced.

Reply:
I tried... but given the dependency on `legacydata.DataTimeRange` it seems ok as a first pass.

Perhaps with this effort we can see what parts of `legacydata` sill need to exist?